src/nvim/os/shell.c

```diff
@@ -418,15 +418,15 @@ static bool out_data_decide_throttle(size_t size)
 
   visit++;
   // Pulse "..." at the bottom of the screen.
-  size_t tick = visit % 4;
-  pulse_msg[0] = (tick > 0) ? '.' : ' ';
-  pulse_msg[1] = (tick > 1) ? '.' : ' ';
-  pulse_msg[2] = (tick > 2) ? '.' : ' ';
+  size_t tick = visit % 3;
+  pulse_msg[1] = (tick > 0) ? '.' : ' ';
+  pulse_msg[2] = (tick > 1) ? '.' : ' ';
   if (visit == 1) {
-    msg_putchar('\n');
+    msg_puts("[throttling output]\n");
```

Comment:
Why? This is tacky, and then also needs translation.

Reply:
It's meant to help with seeing later that output was throttled at the given position.
Any better idea?

Reply:
I guess for the pager that might be useful. What about `[...]` instead?

Reply:
`[...]` is good, added/changed.

Reply:
But it doesn't align with the other dots

Reply:
@justinmk what "other dots" do you mean?

Reply:
the dots printed during the actual pulse. This `[...]` thing shows up and then may be followed by the `.`, `..`, `...` pulse and it looks janky.